Bottom: 79c80554ca72185ed4bcfa02887fb24a98f5755b
Top:    41594d3fca49bd7328fca2bffc72aa44b8bbabdd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 11:39:32 -0500

Refresh of all-docker_image

---
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 575ce46c..be88d7ce 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -61,7 +61,6 @@
 
 - name: build image
   docker_image:
-    force: true
     name: '{{ svn_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-svn'
